static/lang/re-en.json

```diff
@@ -610,6 +610,12 @@
         "Mutagens": {
             "BestialMutagen": {
                 "FeralPrompt": "Increase penalty to AC to increase damage dice size?"
+            },
+            "SanguineMutagen":{
+                "DamageNote":"Whenever you take piercing or slashing damage, you take @Damage[1d6[bleed]].",
+                "LesserVariantSaveLabel":"Sanguine Mutagen (Lesser) vs Disease, Poison, Fatigue",
```

Comment:
Might be able to handle this with a single label:
`"{item} vs Disease, Poison, Fatigue"`